ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ButtonAddCourseToClassroom.vue

```diff
@@ -0,0 +1,114 @@
+<script>
+import utils from 'core/utils'
+import { mapActions } from 'vuex'
+
+export default {
+  props: {
+    course: {
+      type: Object,
+      required: true
+    },
+    classroom: {
+      type: Object,
+      required: true
+    },
+  },
+
+  data () {
+    return {
+      isAddingCourse: false,
+      hasAddedCourse: false,
+    }
+  },
+
+  computed: {
+    isCodeCombat () {
```

Comment:
we do not use this `isCodeCombat` computed value. but I think the JR course is for coco instead of both ? do we want this button to also add other courses? if so, we can clean this code